microsoft/testsuites/acctests/scripts/run_acc_tests.sh

```diff
@@ -0,0 +1,322 @@
+#!/bin/bash
```

Comment:
In LISA, one of advantage is to see raw outputs in log, and raise failure at first place. LISA creates a lot of compatible tools to run with different distros as well.  When scripts is used in tests, it misses those advantages.